README.md

```diff
@@ -1,137 +1,115 @@
 
+ - Stream camera image/video to your computer.
 
-## Upcoming and planned features
+### Upcoming and Planned Features
 
- - Easily monitor print status
+ - Easily monitor print status.
 
- - Integration into other software. Home Assistant? Prusa Slicer?
+ - Integration into other software. Home Assistant? Cura plugin?
 
-Pull requests always welcome :-)
+Let us know what you want to see; Pull requests always welcome! :smile:
 
-## Installation instructions
+## Installation (Prerequisite to the Usage section)
 
-First, please follow the [installation
-instructions](documentation/example-file-usage/example-file-prerequistes.md) for
-your platform.
+Follow the instructions for a [docker install](./documentation/install-from-docker.md) (recommended) or a [git install](./documentation/install-from-git.md).
 
-Verify that you can start `ankerctl.py`, and get the help screen:
+## Usage
 
-### For Windows, use: 
-```powershell
-python3 ankerctl.py -h
-```
-If that does not work then try running using `python ankerctl.py -h`
+### Web Interface
 
-### Linux and MacOS, use:
-```sh
-./ankerctl.py -h
-```
+1. Start the webserver by running one of the following commands in the folder you placed ankerctl in. You’ll need to have this running whenever you want to use the web interface or send jobs to the printer via a slicer:
 
-You should see the below output - if not then head back to [installation instructions](documentation/example-file-usage/example-file-prerequistes.md)
-```
-Usage: ankerctl.py [OPTIONS] COMMAND [ARGS]...
-
-Options:
-  -k, --insecure  Disable TLS certificate validation
-  -v, --verbose   Increase verbosity
-  -q, --quiet     Decrease verbosity
-  -h, --help      Show this message and exit.
-
-Commands:
-  config  View and update configuration
-  http    Low-level http api access
-  mqtt    Low-level mqtt api access
-  pppp    Low-level pppp api access
-```
+   docker:
 
-Before you can use ankerctl, you need to import the configuration.
+   ```sh
+   docker compose up
+   ```
 
-Support for logging in with username and password is not yet supported, but an
-`auth_token` can be imported from the saved credentials found in `login.json` in
-Ankermake Slicer. See `ankerctl.py config import -h` for details:
+   python:
 
-```
-Usage: ankerctl.py config import [OPTIONS] path/to/login.json
+   ```sh
+   ./ankerctl.py webserver run
+   ```
 
-  Import printer and account information from login.json
+2. Navigate to [http://localhost:4470](http://localhost:4470) in your browser of choice on the same computer the webserver is running on. You’ll be prompted to upload your `login.json` file and the given the default path it should be found in your corresponding Operating System. Once the `login.json` has been uploaded, the page will refresh and the web interface is usable. To access it from other devices (must be on the same network), replace “localhost” with the IP address of the computer running the web server.
```

Comment:
Heads up: docker (deliberately) only listens on localhost, so it cannot be accessed from other devices, without some changes.

Reply:
Not clear on what you are requesting here. I can add a note about that if that's what you meant?

Reply:
This part:

> To access it from other devices (must be on the same network), replace “localhost” with the IP address of the computer running the web server.

Is (potentially) wrong. The docker version of the server listens on localhost, and so cannot be accessed (directly) from another computer

Reply:
Are we sure? I just tested on my Mac and can access my container from anywhere within my local network. @spuder / @billyjbryant

Reply:
Removed that sentence